BigQuery/tests/System/ManageTablesTest.php

```diff
@@ -255,6 +255,79 @@ public function testIam()
         $this->assertEquals([$perm], $iam->testPermissions([$perm]));
     }
 
+    public function testSnapshot()
```

Comment:
rename to `testSnapshotAndRestore`

Reply:
Fixed in [59f1f91](https://github.com/googleapis/google-cloud-php/pull/5804/commits/59f1f9111277a7fa3db1f6bec80a7b163a08c8e5)